### DISCUSSION TITLE
Search fails with [HTTP]:500 - ... Version 16.0.0.23821

### DISCUSSION THREAD

**Discussion by RoSchoe** [microsoft-search/pnp-modern-search#3111] [Q&A] [answered]
PNP Search fails on enterprise tenants - we tested it on 3 enterprise tenants (not working) and 2 premium tenants (working)

we currently see no incident from Microsoft on the subject - do you have information on this?

**Version used**
Version 4.9 and 4.8 (we did update to test it in 4.9)

**Describe the bug**
Search fails with [HTTP]:500

Other WebParts which are using pnp search JS with search api fail too with http:500 - we think it is a MS Bug 

**To Reproduce**
use enterprise tenant (e3 / e5)
1. Add PNP Search Box
2. Add PNP Search Results
3. Configure Search results webpart to SharePoint Search and set it together with search box
4. Try search

**Expected behavior**
results are displayed quickly

**Desktop (please complete the following information):**
 - Browser  chrome

**Additional context**
Problem exits since thursday/friday (28/29.06.23)

**Comment by wobba** [comment]
API issues can be logged at https://github.com/SharePoint/sp-dev-docs/issues or direct via Microsoft support.

**Comment by RoSchoe** [comment, marked as answer]
API Fail 500 no longer exists since this morning
